rapidtide/correlate.py

```diff
@@ -626,7 +647,24 @@ def cepstraldelay(data1, data2, timestep, displayplots=True):
     return timestep * np.argmax(residual_cepstrum.real[0 : len(residual_cepstrum) // 2])
 
 
-class aliasedcorrelator:
+class aliasedcorrelator(object):
```

Comment:
I've noticed that there are a fair number of classes and functions that don't really follow PEP8 naming convention. For classes, one would typically use upper camel case (i.e., `AliasedCorrelator`), while for functions one would use snake case (i.e., `aliased_correlator`). Would it be okay for me to try updating some of the class and function names to match that convention?

Reply:
I'm ok with that.  I don't really know anything about PEP8 naming conventions...